### ISSUE TITLE
Search field autocomplete results are not immediately apparent to VoiceOver users [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#250]
Autocomplete suggestions are shown after the search field found on the “Select your county” overlays, with all possible options shown when nothing is entered in the search field. The message “No results found for this search. Try with something else” is shown when there are no results. This message and the fact that there are autocomplete results below the search field is not immediately apparent to VoiceOver users.

![Search field focused with VoiceOver and announced as “Xx”. The user inputted data has found no results.](https://user-images.githubusercontent.com/4437084/92039695-7d5c1580-ed43-11ea-91d1-c032a644f0fe.png)

When there are results shown (initially and when results are found that match the user inputted data) append something like “Showing autocomplete suggestions” to the accessibilityLabel. 

```javascript
<TextInput accessibilityLabel="Search, Showing autocomplete suggestions" />
```

When there are no results, append the “No results …” message to the accessibilityLabel. The visible message text can be then hidden from VoiceOver users, as the text will now be redundant.

```javascript
<TextInput accessibilityLabel="Search, No results found for this search. Try with something else" />

<View accessibilityElementsHidden={true}>
  <Text>No results found for this search. Try with something else</Text>
</View>
```

**Comment by floridemai** [comment]
@willslone This should be addressed in the next build. Not that the search label doesn't update right after you make edits, but the labels and roles should be ok.
